src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.page {
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.calendar {
+    display: flex;
+    flex-wrap: wrap;
+    width: $calendar-width;
```

Comment:
The variable `$calendar-width` is used here but is not defined in this file. Ensure that it is defined or imported from another file.